### ISSUE TITLE
Wallet Explorer

### ISSUE THREAD

**Issue by ghost** [zkSNACKs/WasabiDesign#20]
With the introduction of multi wallet support #8, we have the `Wallet Explorer`.

I think it can be the main way that the user navigates through his wallets. 

![screenshot_20200520_221348](https://user-images.githubusercontent.com/30683012/82492876-383f0b00-9ae7-11ea-84e6-414741a08705.png)

- seeing all the wallets in a nice ordered list
- loading a wallet, this is done easily with a double click, or right click context menu `Load Wallet`.
- doing wallet specific tasks, like testing password [#19] done in a right click context menu
- Navigate the tabs of the wallets, `Send`, `Receive`, `CoinJoin, `History` etc. It is always clear which tab of which wallet is opened, because of the drop down hierarchy
- overview of the wallet balance [of loaded wallets]
- easy activation of lurking wive mode

The `Wallet Manager` was one of the first things added in this new UX redesign, and I think it is very very good. In fact, it has completely changed how I manage the different tabs of the wallet. I no longer click on the tabs in the top for switching the views. Rather, I ALWAYS use the wallet explorer, both to load the wallets, and then to switch the tabs of the wallets.

Further, I have seen multiple users use the new Wasabi, as well as asked multiple users about how they now load wallets and switch between the tabs. And [not so much] surprisingly, the overwhelming majority of users are now using the `Wallet Explorer` to switch the tabs. When I asked users afterwards WHY they used it, they said something along the lines that, "it is a nice overview of all wallets", "it was clear which coinjoin tab of what wallet I'm clicking" etc.

So, both from personal experience, as well as from initial user feedback [though more feedback would be nice], I am confident that the `Wallet Explorer` was a fantastic addition to Wasabi UX for the multi wallet support, and I think that we can leverage it very much in the future design consideration. 

It can be a great tool to declutter the whole experience, for example removing the `Load Wallet` tab #10, or the `Test Password` tab #19.

**Comment by ghost** [comment]
Some improvements that @molnard mentions in #10, and that I personally like a lot too, is to `pin` wallets in the explorer. 

Right now, wallets are ordered based on two criteria...
- loaded wallets are up top
- alphabetical ordered

It would be nice to have this `pined` feature, where a selected wallet is ALWAYS on top, regardless weather it is loaded or what letter it starts with. 

It could be a right click context menu with `Pin Wallet` or `Unpin Wallet`.

Then, the pinned wallets can be ordered based on drag and drop. So, if more than one wallet is pinned, the user can arrange them in whatever order he prefers.

**Comment by kravens** [comment]
Perhaps some extra functions currently only possible by editing the wallet.json file, such as renaming the wallet could be included as right click menu option.

**Comment by ghost** [comment]
Yes @kravens - and these kind of functions should also be available in the "expanded" wallet manager at the launch of Wasabi.
